### PR TITLE
Add zalopay user story

### PR DESCRIPTION
**Pull request by yikeke**

### CODE REVIEW

**Comment by yikeke** [comment]
The title is TBD. But you can take a look at the rest. @CaitinChen

File: zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: 'ZaloPay: Using a Scale-Out MySQL Alternative to Serve Millions of Users'
+author: ['Tan To Nguyen Duy']
+date: 2020-09-17
+summary: 'A DevOps engineer in ZaloPay, Vietnam's most popular mobile payment application, shared why his team chose TiDB as ZaloPay's merchant platform core database. He talked about their pain points, how they use TiDB, and what he likes about it the most.'
+tags: ['HTAP', 'DevCon','TiSpark','Real-time analytics']
```
Comment (CaitinChen) [inline zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md:6]:
```suggestion
tags: ['HTAP', 'DevCon', 'TiSpark', 'Real-time analytics']
```

File: zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+url: /case-studies/zalopay-choosing-a-scale-out-database-to-serve-100-million-users/
+customerCategory: Internet
+categories: ['HTAP']
+logo: /images/blog/customers/vng-logo.png
+image: /images/blog/zalopay-user-story.jpg
+---
+
```
Comment (CaitinChen) [inline zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md:13]:
```suggestion
**Industry:** Mobile Payment

**Author:** Tan To Nguyen Duy (DevOps Engineer at VNG Corporation)
```

File: zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+![ZaloPay User Story](media/zalopay-user-story.jpg)
```
Comment (CaitinChen) [inline zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md]:
```suggestion
![HTAP, MySQL alternative](media/zalopay-user-story.jpg)
```

File: zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+title: 'ZaloPay: Using a Scale-Out MySQL Alternative to Serve Millions of Users'
+author: ['Tan To Nguyen Duy']
+date: 2020-09-17
+summary: A DevOps engineer in ZaloPay, Vietnam's most popular mobile payment application, shared why his team chose TiDB as ZaloPay's merchant platform core database. He talked about their pain points, how they use TiDB, and what he likes about it the most.
+tags: ['HTAP', 'DevCon','TiSpark','Real-time analytics']
+url: /case-studies/zalopay-choosing-a-scale-out-database-to-serve-100-million-users/
+customerCategory: Internet
```
Comment (CaitinChen) [inline zalopay-using-a-scale-out-mysql-alternative-to-serve-millions-of-users.md:8]:
```suggestion
customer: VNG
customerCategory: Financial Services
```

**Pull request merged**